extensions/positron-python/src/client/pythonEnvironments/common/environmentManagers/uv.ts

```diff
@@ -0,0 +1,63 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (C) 2025 Posit Software, PBC. All rights reserved.
+ *  Licensed under the Elastic License 2.0. See LICENSE.txt for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as path from 'path';
+import { pathExists, readFile, resolveSymbolicLink } from '../externalDependencies';
+import { splitLines } from '../../../common/stringUtils';
+import { getPyvenvConfigPathsFrom } from './simplevirtualenvs';
+import { getUserHomeDir } from '../../../common/utils/platform';
+
+/**
+ * Checks if the given interpreter belongs to a uv-managed environment.
+ * If so, the interpreter is expected to have a `pyvenv.cfg` file in the venv folder with a `uv` key
+ * or be located in (or symlinked to) ~/.local/share/uv.
+ * @param interpreterPath Absolute path to the python interpreter.
+ * @returns {boolean} Returns true if the interpreter belongs to a uv environment.
+ */
+export async function isUvEnvironment(interpreterPath: string): Promise<boolean> {
+    const homeDir = getUserHomeDir();
+    // If we can't get the home directory, we can't check if the interpreter is in ~/.local/share/uv,
+    // and it wouldn't be possible to have a valid uv installation in this case anyway
+    if (!homeDir) {
+        return false;
+    }
+
+    // Check if interpreter is directly in the uv directory
+    const uvDir = path.join(homeDir, '.local', 'share', 'uv');
```

Comment:
I think I have to change this to actually running [uv python dir](https://docs.astral.sh/uv/reference/cli/#uv-python-dir) and using that result. It looks like it changes depending on system and environment variables.

Of course this means invocating the `uv` binary, which will add complexity. But it looks like we do this with others like hatch and poetry